cypress/support/fragments/inventory/newInctanceHoldingsItem.js

```diff
@@ -1,111 +1,69 @@
 import uuid from 'uuid';
 import NewInstanceHoldingItem from './holdingsMove/defaultInstanceHoldingItem';
 
-const instanceType = { ...NewInstanceHoldingItem.defaultUiInstanceType };
-const holdingsSources = { ...NewInstanceHoldingItem.defaultUiHoldingsSources };
-const instance = { ...NewInstanceHoldingItem.defaultUiInstance };
-const holding = { ...NewInstanceHoldingItem.defaultUiHolding };
-const materialTypes = { ...NewInstanceHoldingItem.defaultUiMaterialTypes };
-const loanTypes = { ...NewInstanceHoldingItem.defaultUiLoanTypes };
-const createItem = { ...NewInstanceHoldingItem.defaultUicreateItem };
+const instanceType = NewInstanceHoldingItem.defaultUiInstanceType;
```

Comment:
let's use NewInstanceHoldingItem.defaultUiInstanceType into the methods instead of buffer const instanceType